scout/server/blueprints/alignviewers/templates/alignviewers/igv_sashimi_viewer.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta content="Jim Robinson" name="author">
```

Comment:
😸

Reply:
It's only fair I think!

Reply:
I've removed him no since this file is very original from the one I've copied from igv.js